src/views/domain-page/config/domain-page-settings-form.config.ts

```diff
@@ -5,14 +5,14 @@ import { Textarea, SIZE } from 'baseui/textarea';
 import { z } from 'zod';
 
 import { type FormField } from '@/components/form/form.types';
-import formatDurationToDays from '@/utils/data-formatters/format-duration-to-days';
+import formatDurationToSeconds from '@/utils/data-formatters/format-duration-to-seconds';
 
 import DomainPageSettingsRetentionPeriod from '../domain-page-settings-retention-period/domain-page-settings-retention-period';
 import { type DomainInfo } from '../domain-page.types';
 
 export const settingsFormSchema = z.object({
```

Comment:
I initially tried 1) while implementing the form, but I was not able to get type inference working correctly. Would be open to 2) as well but anyone changing the form would have to change both files anyway, so I figured it would be okay to keep both in the same file.

Reply:
yes, 2 can be more work.
Lets discuss the current vs 1

Reply:
In that case, I think it would be best to approach this change in a separate PR